R-package/R/lgb.Booster.R

```diff
@@ -216,7 +216,7 @@ Booster <- R6::R6Class(
       if (!is.null(train_set)) {
 
         # Check if training set is lgb.Dataset
-        if (!lgb.check.r6.class(train_set, "lgb.Dataset")) {
+       if (!lgb.check.r6.class(object = train_set, name = "lgb.Dataset"))
```

Comment:
```suggestion
        if (!lgb.check.r6.class(object = train_set, name = "lgb.Dataset")) {
```

It looks like this `{` was lost, and the lines are no longer aligned.